### ISSUE TITLE
Buiten de lijst om is een beetje zinloos in de meeste gevallen

### ISSUE THREAD

**Issue by thomwiggers** [joostrijneveld/eetFestijn#22]
Buiten de lijst om is een beetje zinloos in de meeste gevallen. Vrijwel altijd wil je het 'via wbw via X' doen. Alleen als degene die haalt niet in wbw staat, moet het anders: dan moet alles cash.

Voorstel: "Buiten de lijst" om wordt een bestel-modus in plaats van een keuze voor iedere individuele persoon, op ongeveer de manier hoe eetvoudig werkt.

**Comment by joostrijneveld** [comment]
Zeker, goed punt. Dit was de makkelijkste manier om te supporten dat een groep die niet/onvoldoende overlapt met de Technicie eetFestijn kan gebruiken zonder in de knoop te komen met de wiebetaaltwat-intergratie.. Maar het kan inderdaad een stuk netter door al eerder die keuze te introduceren. Ik heb een paar keer gespeeld met de gedachte om eetvoudig en eetFestijn meer samen te voegen, maar het is vooral moeite en op dit moment zie ik geen concrete beperkingen.

**Comment by thomwiggers** [comment]
Het grootste probleem is dat je bij bvb een ALV een 50/50 lijst krijgt met mensen die wel/niet op de lijst staan en dan iemand die haalt die niet op de lijst staat – met als gevolg dat het erg verwarrend wordt allemaal.

**Comment by joostrijneveld** [comment]
Right, er ontstaat inderdaad een probleem wanneer degene die haalt niet op de lijst staat, maar er wel mensen aangeven graag via de lijst te willen betalen. Komt dat voor? Waarschijnlijk wel.

**Comment by thomwiggers** [comment]
Zoals gezegd is dat vaak wanneer de site voor een niet-Technicie-scenario gebruikt wordt, zoals bij ALVs.
